prune-objects.js

```diff
@@ -83,20 +99,13 @@ async function pgConnect() {
   return client;
 }
 
-async function allDBKeys(client) {
+async function loadDBKeys(client, table, column) {
   const keys = new Set();
-  const res = await client.query(`
-        select "fileKey" as fk, "rawFileKey" as rk from "Recordings"
-        union
-        select "fileKey" as fk, NULL as rk from "Files"
-  `);
+  const res = await client.query(
+    `select "${column}" as k from "${table}" where 1 is not NULL`
```

Comment:
why the where clause?

Reply:
`1` refers to the first (and only) column. This just removes the rows where the field is not set.